build/web_compose/Dockerfile

```diff
@@ -118,12 +118,12 @@ COPY import/. /workspace/import/
 COPY tools/nl/embeddings/. /workspace/tools/nl/embeddings/
 
 # Download model and embeddings
-ENV IS_CUSTOM_DC=true
-RUN python -m tools.nl.embeddings.build_custom_dc_embeddings --mode=download
+RUN mkdir -p /tmp/datcom-nl-models
+RUN gsutil cp -R gs://datcom-nl-models/ft_final_v20230717230459.all-MiniLM-L6-v2/ /tmp/datcom-nl-models/
+RUN gsutil cp gs://datcom-nl-models/embeddings_medium_2024_05_09_18_01_32.ft_final_v20230717230459.all-MiniLM-L6-v2.csv /tmp/datcom-nl-models/
```

Comment:
since embeddings version can change, should we be reading this from embeddings.yaml instead?